src/Tooltip/tooltip.css

```diff
@@ -31,19 +20,22 @@
     max-width:      100%;
 }
 
-.tooltip
+
+.tooltip,
+.tooltipDismissible
 {
-    display:            block;
+    display:            flex;
+    align-items:        start;
```

Comment:
tbh I’m not very clear on the difference between `start`, `flex-start` and `self-start`... but for consistency with what we have been using so far, I think we should stick to `flex-start`